### PR TITLE
Flatten 1010cg UI schema

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
To make our application more readable we are un-nesting the UISchema and breaking it up into files based on sharedUI, caregiverUI, and veteranUI

[Ticket](https://app.zenhub.com/workspaces/vsa---caregiver-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/16874)

## Testing done
Covered by e2e and Unit Teat

## Acceptance criteria
- [x] Form works exactly as before
- [x] UISchema is flat
- [x] UISchema is broken into separate files based on category
- [x] Constants are no longer nested

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/caregivers/definitions/UIDefinitions/sharedUI.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      },
+    },
+  ],
+});
+
+export const addressWithoutCountryUI = label => ({
+  'ui:title': ' ',
+  'ui:order': ['street', 'street2', 'city', 'state', 'postalCode'],
+  street: {
+    'ui:title': `${label}  current street address`,
```
Comment (Mottie) [inline src/applications/caregivers/definitions/UIDefinitions/sharedUI.js:99]: Not a big deal, but it looks like every `${label}` in this file is followed by two spaces.

**Review by Mottie — APPROVED** [review]
ghe-emoji

File: src/applications/caregivers/definitions/UIDefinitions/sharedUI.js (hunk 1 above)
Comment (LinnJS) [reply src/applications/caregivers/definitions/UIDefinitions/sharedUI.js:99]: Awesome catch would not have caught this one in a million years! 🚀

Comment (Mottie) [reply src/applications/caregivers/definitions/UIDefinitions/sharedUI.js]: In the Engineering all-hands on Friday, I'll talk about userscripts... check this one out - https://github.com/Mottie/GitHub-userscripts/wiki/GitHub-code-show-whitespace

**Review by craigwheeler — APPROVED** [review]

**Comment by ala-yna** [comment]
Hey @LinnJS - I looked through the RI and compared it to what we have in Production. LGTM.

**Pull request merged**